packages/core/src/components/Dialog/Dialog.jsx

```diff
@@ -83,6 +103,10 @@ Dialog.propTypes = {
    * at the bottom of the dialog.
    */
   actions: PropTypes.node,
+  /**
+   * Additional classes to be added to the actions element.
```

Comment:
✍️ Nit: Replace `element` with `container` since the class is applied to the node that wraps whatever is passed in as the `actions` prop